VERSION

```diff
@@ -0,0 +1,2 @@
+# This file is auto-generated by a GitHub Action -- do not edit
```

Comment:
should this file be checked in?

Reply:
Good question! Yeah it should actually. This is for a kind of unusual case tbh, but it's for any scenario where the code gets downloaded without the git metadata, then the build will still work by looking up the version from this file.

Reply:
Yeah I think that would be a good idea! I'd rather have one more release where it's just adding these updated Python bindings. But after that a 1.0.0 is probably a good idea.